src/Components/ContentProperty.tsx

```diff
@@ -1,9 +1,20 @@
 import * as React from "react";
 import * as Scrivito from "scrivito";
 
+type ValidationResultObject = {
+  severity?: string | undefined;
+  message: string;
+};
+type ContentPropertyProps = {
```

Comment:
I would use an interface for defining the types of objects. It is common practice and offers several advantages, such as extendibility.

Reply:
Yes, you are right.
Done.